docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+      - [`targetGas`](#targetgas)
+      - [`baseFeeChangeDenominator`](#basefeechangedenominator)
+      - [`minBlockGasCost`](#minblockgascost)
+      - [`maxBlockGasCost`](#maxblockgascost)
+      - [`blockGasCostStep`](#blockgascoststep)
+    - [Validator Fee Recipient](#validator-fee-recipient)
+    - [Precompiles](#precompiles)
+      - [`contractDeployerAllowListConfig`](#contractdeployerallowlistconfig)
+      - [`contractNativeMinterConfig`](#contractnativeminterconfig)
+      - [`txAllowListConfig`](#txallowlistconfig)
+    - [Genesis Block](#genesis-block)
+      - [`nonce`](#nonce)
+      - [`timestamp`](#timestamp)
+      - [`extraData`](#extradata)
+      - [`gasLimit`](#gaslimit-1)
+      - [`difficulty`](#difficulty)
+      - [`mixHash`](#mixhash)
+      - [`coinbase`](#coinbase)
+      - [`parentHash`](#parenthash)
+      - [`gasUsed`](#gasused)
+      - [`number`](#number)
+      - [`airdropHash` `baseFeePerGas` `airdropAmount`](#airdrophash-basefeepergas-airdropamount)
+    - [Native Token Allocation](#native-token-allocation)
+- [Creating a Custom Genesis File](#creating-a-custom-genesis-file)
+  - [Setting the Chain Id](#setting-the-chain-id)
+  - [Configuring Fees and Gas](#configuring-fees-and-gas)
+  - [Allocating Native Tokens](#allocating-native-tokens)
+  - [Minting Native Tokens](#minting-native-tokens)
+  - [Restricting Contract Deployers](#restricting-contract-deployers)
+- [Creating and Deploying the Subnet Using the Custom Genesis File](#creating-and-deploying-the-Subnet-using-the-custom-genesis-file)
+- [Deploying the Game Contract](#deploying-the-game-contract)
+  - [Metamask Configuration](#metamask-configuration)
+  - [Metamask Gas Configuration](#metamask-gas-configuration)
+  - [Deploying the Contract](#deploying-the-contract)
+    - [NativeMinterInterface.sol](#nativeminterinterfacesol)
+    - [Game.sol](#gamesol)
+    - [Compiling and deploying](#compiling-and-deploying)
+  - [Setting Game Contract as a Minter](#setting-game-contract-as-minter)
+  - [Play the Game](#play-the-game)
+- [Conclusion](#conclusion)
+  - [Resources](#resources)
+
+## Prerequisites
+
+### Metamask
+You need to have [Metamask](https://metamask.io/) extension installed on your browser.
+
+### Avalanche CLI
+
+Create a folder for Avalanche CLI.
+
+```bash
+$ mkdir -p ~/avalanche
+```
+
+Go to that directory and download the Avalanche CLI.
+
+```bash
+$ cd ~/avalanche
+$ curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-cli/main/scripts/install.sh | sh -s
+
+> ava-labs/avalanche-cli info checking GitHub for latest tag
+> ava-labs/avalanche-cli info found version: 0.1.3 for linux/amd64
+> ava-labs/avalanche-cli info installed ./bin/avalanche
+```
+
+Go to the directory where you just downloaded the CLI and add it to your PATH. So you can run the CLI from anywhere.
+
+```bash
+$ cd ./bin/avalanche
+$ export PATH=$PWD:$PATH
+```
+
+We're ready to deploy our first Subnet.
+
+# Creating and deploying a Simple Subnet
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) You shouldn't have multiple `<h1>`s in the same document.

> MD025/single-title/single-h1: Multiple top-level headings in the same document

[More info: MD025](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document)